.github/workflows/scripts/detect_ts_ignore.py

```
@@ -0,0 +1,86 @@
+        int: 0 if no violations, 1 if violations are found.
+    """
+    ts_ignore_found = False
+
+    for file in files:
+        try:
+            logging.info(f"Checking file: {file}")
+            with open(file, encoding="utf-8") as f:
+                for line_num, line in enumerate(f, start=1):
+                    # Handle more variations of @ts-ignore
+                    if re.search(
+                        r"(?://|/\*)\s*@ts-ignore(?:\s+|$)", line.strip()
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)